go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	k8s.io/api v0.29.1
 	k8s.io/apimachinery v0.29.1
 	k8s.io/client-go v0.29.1
-	k8s.io/cri-api v0.29.1
+	k8s.io/cri-api v0.30.0-alpha.2.0.20240217224521-840a52e4cd66
 	k8s.io/klog/v2 v2.120.1
 	k8s.io/kubelet v0.29.1
```

Comment:
Can we bump all of them to [v1.30.0-alpha.2](https://github.com/kubernetes/kubernetes/tree/v1.30.0-alpha.2)?